tox.ini

```diff
@@ -2,12 +2,16 @@
 envlist =
        {py38,py39,py310}-django32-drf3.14-df22.1,
        {py38,py39}-{django41,django42}-drf3.14-df22.1,
-       {py310,py311}-{django41,django42,djangomain}-drf3.14-df22.1,
+       {py310,py311}-{django41,django42,django50,djangomain}-drf3.14-df22.1,
+       py312-{django50,djangomain}-drf3.14-df22.1,
        py311-lint
 
 [testenv]
 commands =
-       python -W error::DeprecationWarning -W error::PendingDeprecationWarning \
+       python \
+       -W error::DeprecationWarning \
+       -W error::PendingDeprecationWarning \
+       -W "once:Converter 'drf_format_suffix' is already registered:" \
```

Comment:
That's a warning that shows up when running against Django main. I've switched the filter to log it once for now. I'm not too sure if the fix belongs here or in DRF to be honest.